js/src/get-started-page/case-studies-card/index.scss

```diff
@@ -0,0 +1,53 @@
+.gla-get-started-case-studies-card {
+	// Adjust <CardHeader> imported from @wordpress/components.
+	// Repeat selector to make it higher priority.
+	.components-card__header.components-card__header {
+		flex-direction: column;
+		padding: $grid-unit * 5.75 $grid-unit * 14 $grid-unit-50;
+		text-align: center;
+
+		@media (max-width: $break-small) {
+			padding: $grid-unit-30;
+		}
+	}
+
+	.components-flex {
+		gap: $grid-unit * 3.5;
+		padding: 0 $grid-unit * 7.75 $grid-unit * 5.75;
```

Comment:
💅 / ❓ 

Not sure if this is already discussed but maybe we can create some SCSS variables for the fonts and some spacing is being reused all over `get-started-page` directory.

Reply:
Thanks for the suggestion! I added a commit 13e1ac1 to extract some shared variables for font sizes and line heights as they've got more common uses. 

There are too many different sizes of the spacing that I've used in get started page based on the design, for example:

```
10px;
20px;
28px;
40px;
44px;
46px;
50px;
55px;
62px;
88px;
112px;
```

It's a bit tricky to make them as variables tho.